### ISSUE TITLE
Problem with checking if point is visible in the current image in DepthFilter::updateSeeds

### ISSUE THREAD

**Issue by tanmengwen** [uzh-rpg/rpg_svo#104]
https://github.com/uzh-rpg/rpg_svo/blob/master/svo/src/depth_filter.cpp#L228
shouldn't it be like this ?
if(!frame->cam_->isInFrame(frame->f2c(xyz_f).cast<int>())) {
      ++it; // point does not project in image
      continue;
}

The original code is checking the point in the reference keyframe image, however the point has been transformed to the current frame coordination.

**Comment by cfo** [comment]
yes, you are correct. can you change it and send a pull request?
thanks!
